test failure:
```
FAIL: test_all_media_paths_with_inline_video (corehq.apps.app_manager.tests.test_media_suite.MediaSuiteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/app_manager/tests/test_media_suite.py", line 49, in test_all_media_paths_with_inline_video
    self.assertEqual(app.all_media_paths, [inline_video_path])
AssertionError: set(['jr://file/commcare/video-inline/data/inline_video.mp4']) != ['jr://file/commcare/video-inline/data/inline_video.mp4']
```

--- a/corehq/apps/app_manager/tests/test_media_suite.py
+++ b/corehq/apps/app_manager/tests/test_media_suite.py
@@ -46,7 +46,7 @@ class MediaSuiteTest(SimpleTestCase, TestXmlMixin):
         app = Application.wrap(self.get_json('app_video_inline'))
 
         self.assertTrue(app.get_module(0).uses_media())
-        self.assertEqual(app.all_media_paths, [inline_video_path])
+        self.assertEqual(app.all_media_paths, set([inline_video_path]))
 
     @override_settings(BASE_ADDRESS='192.cc.hq.1')
     def test_case_list_media(self):

